code/game/gamemodes/cult/cult_objectives.dm

```diff
@@ -155,12 +155,12 @@
 /datum/game_mode/cult/proc/get_possible_sac_targets()
 	var/list/possible_sac_targets = list()
 	for(var/mob/living/carbon/human/player in GLOB.player_list)
-		if(player.mind && !is_convertable_to_cult(player.mind) && (player.stat != DEAD))
+		if(player.mind && !is_convertable_to_cult(player.mind) && (player.stat != DEAD) && (player.mind.offstation_role != TRUE) )
```

Comment:
Not the proper check. This is a boolean, so explicitly checking for a specific value is unnecessary.

Reply:
I put it this way because I recall one time I was trying something without the specific value and it didn't work, so did aa but I'll change it. I'll change it anyways I guess